workflow-templates/phpunit-mysql.yml

```diff
@@ -0,0 +1,113 @@
+name: PHPUnit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+env:
+  # Location of the phpunit.xml and phpunit.integration.xml files
+  PHPUNIT_CONFIG: ./tests/phpunit.xml
+  PHPUNIT_INTEGRATION_CONFIG: ./tests/phpunit.integration.xml
+
+jobs:
+  phpunit-mysql:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # do not stop on another job's failure
```

Comment:
why not? in most cases it doesn't add much value to have the full matrix fail. as soon as one job fails you need to fix something. Redundant failing tests use CI time and resources that block other jobs.

Reply:
Yeah I think it would make sense to remove the fail-fast: false to have shorter queue in CI. I can just remove the line and the comment I suppose, it defaults to true?

Reply:
Is this globally or per "file".
I'm totally fine with killing other phpunit-mysql
I'm little bit fine with killing other phpunit
I'm not so fine with killing php-lint, cs, node, ...

While you still need to repush after fixing e.g. phpunit-mysql it's good to know linting and other results up front as well, reducing the round trips for the developer

Reply:
We’ll see, it can come back later if it’s a pain. Removing this for now.

Reply:
It is per file it seems.